tiledb/sm/filesystem/azure.h

```diff
@@ -437,13 +451,40 @@ class Azure {
     return AzureScanner<F, D>(*this, parent, f, d, recursive, max_keys);
   }
 
+  /**
+   * Creates a directory.
+   *
+   * @param uri The directory's URI.
+   */
+  void create_dir(const URI&) const {
+    // No-op. Stub function for other filesystems.
+  }
+
+  /**
+   * Copies the directory at 'old_uri' to `new_uri`.
+   *
+   * @param old_uri The directory's current URI.
+   * @param new_uri The directory's URI to move to.
+   */
+  void copy_dir(const URI&, const URI&) const {
```

Comment:
This could be implemented by listing and copying all blobs; I guess it's not currently implemented for Azure?